scripts/fast-reboot

```diff
@@ -69,7 +73,7 @@ sonic_asic_type=$(sonic-cfggen -y /etc/sonic/sonic_version.yml -v asic_type)
 function clear_warm_boot()
 {
     debug "Failure ($?) cleanup ..."
-    config warm_restart disable || /bin/true
+    timeout 10s config warm_restart disable || /bin/true
```

Comment:
Do you mind to add a check here and if timeout happened, it prints an error message?

Reply:
That wasn't easy. But I got it figured out :-)